lib/rust/ensogl/example/slider/src/lib.rs

```diff
@@ -24,20 +24,286 @@
 use ensogl_core::prelude::*;
 use wasm_bindgen::prelude::*;
 
+use enso_frp as frp;
+use ensogl_core::application::shortcut;
 use ensogl_core::application::Application;
+use ensogl_core::application::View;
 use ensogl_core::data::color;
-use ensogl_core::display::object::ObjectOps;
+use ensogl_core::display;
 use ensogl_hardcoded_theme as theme;
 use ensogl_slider as slider;
 use ensogl_text_msdf::run_once_initialized;
 
 
 
+// ===================================
+// === Basic slider initialization ===
+// ===================================
+
+/// Create a basic slider.
+fn make_slider(app: &Application) -> slider::Slider {
+    let slider = app.new_view::<slider::Slider>();
+    slider.frp.set_background_color(color::Lcha(0.8, 0.0, 0.0, 1.0));
+    slider.frp.set_max_value(5.0);
+    slider.frp.set_default_value(1.0);
+    slider.frp.set_value(1.0);
+    slider
+}
+
+
+
+// ========================
+// === Model definition ===
+// ========================
+
+/// The slider collection model holds a set of sliders that can be instantiated and dropped.
+#[derive(Debug, Clone, CloneRef)]
+pub struct Model {
+    /// Vector that holds example sliders until they are dropped.
+    sliders: Rc<RefCell<Vec<slider::Slider>>>,
+    app:     Application,
+    root:    display::object::Instance,
+}
+
+impl Model {
+    fn new(app: &Application) -> Self {
+        let app = app.clone_ref();
+        let sliders = Rc::new(RefCell::new(Vec::new()));
+        let root = display::object::Instance::new();
+        let model = Self { app, sliders, root };
+        model.init_sliders();
+        model
+    }
+
+    /// Add example sliders to scene.
+    fn init_sliders(&self) {
+        let slider1 = make_slider(&self.app);
+        slider1.frp.set_width(400.0);
+        slider1.frp.set_height(50.0);
+        slider1.set_y(-120.0);
+        slider1.frp.set_value_indicator_color(color::Lcha(0.4, 0.7, 0.7, 1.0));
```

Comment:
Nothing to change here, but when writing tests, its sometimes just easier to write `color::Lcha::from(color::Rgba(...))`. It's up to you which one you chose in tests. Im writing this only to let you know :)